xml/journalctl.xml

```diff
@@ -372,7 +372,7 @@ apache2.service - The Apache Webserver
 TTYPath=/dev/tty12</screen>
   </sect2>
 
-  <sect2 xml:id="journalctl.config.forwardtosyslog">
+  <sect2 xml:id="sec.journalctl.config.forwardtosyslog">
```

Comment:
"forwardtosyslog" is long and hard to read. how about just "syslog"?
is "config" really necessary?